pytao/interface_commands.py

```diff
@@ -831,17 +874,33 @@ def datum_create(tao, *, datum_name, data_type, ele_ref_name="", ele_start_name=
     s_offset : default=0
     ix_bunch : default=0
     invalid_value : default=0
-    spin_axis%n0(1) : optional
-    spin_axis%n0(2) : optional
-    spin_axis%n0(3) : optional
-    spin_axis%l(1) : optional
-    spin_axis%l(2) : optional
-    spin_axis%l(3) : optional
+    spin_axis%n0(1) : optional, default=''
+    spin_axis%n0(2) : optional, default=''
+    spin_axis%n0(3) : optional, default=''
+    spin_axis%l(1) : optional, default=''
+    spin_axis%l(2) : optional, default=''
+    spin_axis%l(3) : optional, default=''
     
```

Comment:
If default is empty just leave as `optional`.

Reply:
I changed things and corrected this.